<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFRDRImagingFCSCorrelationDialog</class>
 <widget class="QWidget" name="QFRDRImagingFCSCorrelationDialog">
  <property name="modal" stdset="0">
   <bool>false</bool>
  </property>
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>imFCS: Correlator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qfrdrimagingfcs.qrc">
    <normaloff>:/imaging_fcs/qfrdrimagingfcs_correlate.png</normaloff>:/imaging_fcs/qfrdrimagingfcs_correlate.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string> New Correlation Job: </string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;b&gt;&amp;Image File:&lt;/b&gt;</string>
        </property>
        <property name="buddy">
         <cstring>edtImageFile</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="edtImageFile"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnSelectImageFile">
          <property name="toolTip">
           <string>select an image series file to correlate</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="qfrdrimagingfcs.qrc">
            <normaloff>:/imaging_fcs/fileopen.png</normaloff>:/imaging_fcs/fileopen.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnLoad">
          <property name="toolTip">
           <string>read metadata from the given image series file</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="qfrdrimagingfcs.qrc">
            <normaloff>:/imaging_fcs/use_file.png</normaloff>:/imaging_fcs/use_file.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labDetails">
        <property name="text">
         <string>&lt;b&gt;Details:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="widDetails">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>file &amp;format:</string>
           </property>
           <property name="buddy">
            <cstring>cmbFileformat</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QComboBox" name="cmbFileformat">
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>none</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>TIFF</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>&amp;prefix for output:</string>
             </property>
             <property name="buddy">
              <cstring>edtPrefix</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edtPrefix">
             <property name="text">
              <string>./results/</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>frame &amp;timing:</string>
           </property>
           <property name="buddy">
            <cstring>edtFrameTime</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>frame &amp;time:</string>
             </property>
             <property name="buddy">
              <cstring>edtFrameTime</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="JKDoubleEdit" name="edtFrameTime">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Âµs</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>80</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>frame &amp;rate:</string>
             </property>
             <property name="buddy">
              <cstring>edtFrameRate</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="JKDoubleEdit" name="edtFrameRate">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>fps</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>&amp;correlator:</string>
           </property>
           <property name="buddy">
            <cstring>cmbCorrelator</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QComboBox" name="cmbCorrelator">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
               </property>
               <item>
                <property name="text">
                 <string>Direct Correlation</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Multi-Tau 1 (JanK)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Multi-Tau 2 (JanB)</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string># Correlators (&amp;S):</string>
               </property>
               <property name="buddy">
                <cstring>spinS</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinS">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="value">
                <number>18</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string># Lags/Cor. &amp;(P):</string>
               </property>
               <property name="buddy">
                <cstring>spinP</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinP">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="value">
                <number>16</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string># lag factor (&amp;m):</string>
               </property>
               <property name="buddy">
                <cstring>spinM</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinM">
               <property name="minimum">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labCorrelator">
               <property name="text">
                <string>&amp;tau;&lt;sub&gt;min&lt;/sub&gt;=1</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>&amp;background correction:</string>
           </property>
           <property name="buddy">
            <cstring>cmbBackground</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QComboBox" name="cmbBackground">
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>none</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>remove offset</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>remove minimum counts &amp; offset</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>subtract &amp;offest:</string>
             </property>
             <property name="buddy">
              <cstring>edtOffset</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="JKDoubleEdit" name="edtOffset">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>&amp;add to project:</string>
           </property>
           <property name="buddy">
            <cstring>chkAddToProject</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QCheckBox" name="chkAddToProject">
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnAddJob">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>execute the correlation job, as defined above</string>
             </property>
             <property name="text">
              <string>&amp;add job</string>
             </property>
             <property name="icon">
              <iconset resource="qfrdrimagingfcs.qrc">
               <normaloff>:/imaging_fcs/add_job.png</normaloff>:/imaging_fcs/add_job.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string> Progress: </string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="layProgress_3" stretch="0,0,1">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <widget class="QSpinBox" name="spinProcesses">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>max. parallel &amp;jobs:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="buddy">
           <cstring>spinProcesses</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Overall Progress:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <widget class="QWidget" name="widProgress2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>838</width>
           <height>69</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="layProgress">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JKDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>jkdoubleedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtImageFile</tabstop>
  <tabstop>btnSelectImageFile</tabstop>
  <tabstop>btnLoad</tabstop>
  <tabstop>edtFrameTime</tabstop>
  <tabstop>edtFrameRate</tabstop>
  <tabstop>cmbCorrelator</tabstop>
  <tabstop>spinS</tabstop>
  <tabstop>spinP</tabstop>
  <tabstop>spinM</tabstop>
  <tabstop>cmbBackground</tabstop>
  <tabstop>edtOffset</tabstop>
  <tabstop>chkAddToProject</tabstop>
  <tabstop>btnAddJob</tabstop>
  <tabstop>spinProcesses</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="qfrdrimagingfcs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QFRDRImagingFCSCorrelationDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>719</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
